blazor/dropdown-list/selection.md

```diff
@@ -0,0 +1,147 @@
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/getting-started/get-selected-value.razor %}
+
+{% endhighlight %}
+
+* You can get the complete object list of the selected value from the **ItemData** property in the [ValueChange event argument](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.ChangeEventArgs-2.html#Syncfusion_Blazor_DropDowns_ChangeEventArgs_2_ItemData).
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/getting-started/item-data.razor %}
+
+{% endhighlight %}
+
+## Preselected Value on OnInitializedAsync
+
+You can bind pre-selected value to the DropDownList component using `@bind-Value` property. We can assign the value property inside `OnInitializedAsync` lifecycle. The below sample demonstrates how to bind the value on initial rendering of the component.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/preselect-value.razor %}
+
+{% endhighlight %}
+
+The output will be as follows,
+
+![Blazor DropDownList with pre-select value](./images/selection/blazor_dropdown_preselect-value.png)
+
+## Programmatically Change the Selected Value
+
+You can change the component value programmatically or externally by the component instance using `@ref` property of the component. The below sample demonstrates how to change the value of the component using click event of the button component.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/change-the-selected-value.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with pre-select value before](./images/selection/blazor_dropdown_changing-selected-value.gif)
+
+## Preselect Value with Index
+
+You can bind pre-selected value to the component also using [@bind-Index](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Index) property. It binds the respective value present in the binded index.
+
+> It will dependent on [SortOrder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SortOrder.html) type. It will bind the value to the component with sorted data if the corresponding property is defined.
+The below sample demonstrates how to bind the index on initial rendering.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/preselect-value-index.razor %}
+
+{% endhighlight %}
+
+The output will be as follows,
+
+![Blazor DropDownList with bind-index](./images/selection/blazor_dropdown_preselect-value-index.png)
+
+## Get Selected Item by Value
+
+You can get all the datasource items belongs to the value selected in the component by using [GetDataByValue](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_GetDataByValue__0_) method. It will returns the items with the return type of TValue mapped to the component. 

```

Comment:
Remove this - It will returns the items with the return type of TValue mapped to the component.